sdk/confidentialledger/azure-confidentialledger/azure/confidentialledger/_patch.py

```diff
@@ -75,6 +78,17 @@ def __init__(
         ledger_certificate_path: Union[bytes, str, os.PathLike],
         **kwargs: Any,
     ) -> None:
+        # Remove some kwargs first so that there aren't unexpected kwargs passed to
+        # get_ledger_identity.
+        if isinstance(credential, ConfidentialLedgerCertificateCredential):
+            auth_policy = None
+        else:
+            credential_scopes = kwargs.pop("credential_scopes", ["https://confidential-ledger.azure.com/.default"])
```

Comment:
Do you have multi-cloud support?

Are they using same scope?

Reply:
We do not have multi-cloud support